index.html

```diff
@@ -17,7 +17,7 @@
     <!-- TODO put ellipsis in :after -->
 
     <header>
-      <button class="create-album btn primary">Create album +</button>
```

Comment:
I'd like to keep the "Create album" button in the header.  I like having a header in each view that has a primary action button in the same spot.